scopes/lanes/lanes/lanes.main.runtime.ts

```diff
@@ -369,8 +374,12 @@ export class LanesMain {
     return lane;
   }
 
-  async removeLanes(laneNames: string[], { remote, force }: { remote: boolean; force: boolean }): Promise<string[]> {
-    const results = await removeLanes(this.workspace?.consumer, laneNames, remote, force);
+  async removeLanes(laneNames: string[], opts?: { remote: boolean; force: boolean }): Promise<string[]> {
+    if (!this.workspace) {
+      await this.scope.legacyScope.lanes.removeLanes(this.scope.legacyScope, laneNames, true);
```

Comment:
Maybe you want to check also that `--remote` is false, because this API also allows deleting *remote* lanes without workspace